src/views/Log.vue

```diff
@@ -474,10 +478,44 @@ export default {
         }
       }
       return this.workflowId
+    },
+    controlGroups () {
+      return [
+        {
+          title: 'Log',
+          controls: [
+            {
+              title: 'Timestamps',
+              icon: mdiClockOutline,
+              action: 'toggle',
+              value: this.timestamps,
+              key: 'timestamps'
+            },
+            {
+              title: 'Word wrap',
+              icon: mdiWrap,
+              action: 'toggle',
+              value: this.wordWrap,
+              key: 'wordWrap',
+            },
+            {
+              title: 'Auto scroll',
+              icon: mdiMouseMoveDown,
+              action: 'toggle',
+              value: this.autoScroll,
+              key: 'autoScroll',
+            },
+          ]
+        }
+      ]
     }
   },
 
   methods: {
+    scrollToTop () {
+      this.autoScroll = false
+      this.logScrollEl.scrollTo(0, 0)
+    },
```

Comment:
Would be good to keep this alongside the rest of the autoscroll code inside `setup`